### DISCUSSION TITLE
Will my Github Education pack lose when I graduate

### DISCUSSION THREAD

**Discussion by neverbiasgit** [community/community#136205] [GitHub Education] [answered]
And if I continue postgraduate, will I get it again?

**Comment by neverbiasu** [comment, marked as answer]
Student benefits last for 2 years. And the [docs ](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student)of it is below.
[Expiration and renewals](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student#expiration-and-renewals)
Once your GitHub Education access expires, you may reapply if you're still eligible, although some of our partner offers for GitHub Student Developer Pack cannot renew. Most of the timed offers from our partners start once you set them up. To reapply, simply return to https://education.github.com/, click your profile picture, then click Reverify your academic affiliation.

Screenshot of a profile menu on the GitHub Education website. The "Reverify your academic affiliation" menu option is outlined in dark orange.

For more information, see the [GitHub Student Developer Pack](https://education.github.com/pack) page.

To see when your free access to the GitHub Student Developer Pack expires, visit your account's [billing settings](https://github.com/settings/billing).

**Comment by berthablea** [comment]
I think so.
https://orlandojiujitsu.com/birthday-parties/
